models/engine/file_storage.py

```diff
@@ -70,4 +70,20 @@ def close(self):
         self.reload()
 
     def get(self, cls, id):
-        """Returns the object based on the class name and its ID"""
+        """ method to retrieve one object """
+        if cls:
+            for key, value in self.__objects.items():
+                if value.id == id:
+                    return (value)
+        else:
+            return None
+
+    def count(self, cls=None):
+        """method to count the number of objects in storage"""
+        num_objects = 0
```

Comment:
num_objects is a good variable name because is just for the count the total number of objects that has the JSON file